src/index-internal.ts

```diff
@@ -1,18 +1,18 @@
 // Until we determine how to use barrels with the Aurelia-CLI,
 // use this internal index only with the tests.
-import IredirectHandler from "./iredirect-handler";
-import OpenIdConnectAuthorizeStep from "./open-id-connect-authorize-step";
-import OpenIdConnectConfigurationManager from "./open-id-connect-configuration-manager";
-import OpenIdConnectFactory from "./open-id-connect-factory";
-import OpenIdConnectLogger from "./open-id-connect-logger";
-import OpenIdConnectNavigationStrategies from "./open-id-connect-navigation-strategies";
-import OpenIdConnectRouting from "./open-id-connect-routing";
-import OpenIdConnectUserBlock from "./open-id-connect-user-block";
-import OpenIdConnectUserDebug from "./open-id-connect-user-debug";
+import { RedirectHandler } from './redirect-handler';
```

Comment:
I think this change reflects the canonical linting rules/style that Aurelia uses for TypeScript. Is that right?

Reply:
Yes since this its carrying 'aurelia' in its name I applied the same coding conventions.
In typescript the difference between a interface and a class is smaller then for example in .NET.
Prefixing interfaces with an I is less common typescript.